sftpcloudfs/main.py

```diff
@@ -416,10 +415,16 @@ def run(self):
         dc.pidfile = self.pidfile
 
         if self.options.uid:
-            dc.uid = self.options.uid
+            if isinstance(self.options.uid, basestring):
+                dc.uid = pwd.getpwnam(self.options.uid).pw_uid
+            else:
+                dc.uid = self.options.uid
 
         if self.options.gid:
-            dc.gid = self.options.gid
+            if isinstance(self.options.gid, basestring):
+                dc.gid = pwd.getpwnam(self.options.gid).pw_gid
```

Comment:
This can raise a `KeyError`. We should capture that and provide an user friendly error.

Reply:
Same as `self.options.uid`